ui/app/AppLayouts/Chat/ChatLayout.qml

```diff
@@ -96,14 +174,14 @@ StackLayout {
         id: joinCommunityViewComponent
         JoinCommunityView {
             id: joinCommunityView
-            readonly property var communityData: sectionItemModel
+            readonly property var communityData: sectionItemWithMembers
             readonly property string communityId: communityData.id
             name: communityData.name
             introMessage: communityData.introMessage
             communityDesc: communityData.description
             color: communityData.color
             image: communityData.image
-            membersCount: communityData.members.count
+            membersCount: communityData.joinedMembers.count
```

Comment:
Will this even work? The `communityData` is the `sectionItemWithMembers` copy but you defined it as:
```qml
sectionItemCopy.members = joinedMembers
```
So here, `joinedMembers` should not be available